packages/components/grid/src/grid.scss

```diff
@@ -2,7 +2,7 @@
 :host {
```

Comment:
Isn't the grid header background too dark? Previously it was `#f6f6f6` I guess?

<img width="2102" alt="image" src="https://github.com/user-attachments/assets/e3862213-eb71-453c-babc-c5df87456fa1">

Reply:
Fixed